src/internal.c

```diff
@@ -21029,7 +21086,10 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
         }
         #endif /* WOLFSSL_DTLS */
 
-        /* check for available size */
+        if (IsEncryptionOn(ssl, 1))
+            sendSz += MAX_MSG_EXTRA;
+
+        /* check for avalaible size */
```

Comment:
"available"

Reply:
Changes made. Will write up a doc on running secure renegotiation using the example server and client.